recipes/drms/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "drms" %}
+{% set version = "0.5.2" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "05147964834b1f24730c824addc6c2160dba0953beda26234113740ace3378fa" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+  noarch: python
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy >=1.9.0
+    - pandas >=0.14.1
+    - six >=1.8.0
+
+test:
+  imports:
+    - drms
+
+about:
+  home: https://github.com/kbg/drms
+  license: MIT
+  license_family: MIT
+  license_file: 'LICENSE.txt'
+  summary: Access HMI, AIA and MDI data with Python
+  description: |
+    The ``drms`` module provides an easy-to-use interface for accessing HMI, AIA and MDI data with Python.
+    It uses the publicly accessible `JSOC <http://jsoc.stanford.edu/>`_ DRMS server by default, but can also\nbe used with local `NetDRMS <http://jsoc.stanford.edu/netdrms/>`_ sites.
+    More information, including a detailed tutorial is available on `Read the Docs <http://drms.readthedocs.io/>`_.
+  doc_url: http://drms.readthedocs.io/
+  dev_url: https://github.com/kgb/drms
+
+extra:
+  recipe-maintainers:
+    - SolarDrew
+    - Cadair
+    - nabobalis
```

Comment:
Now the question is whether @nabobalis is okay with this

Reply:
I assume so if Cadair says to add him - @nabobalis speak now or forever etc.

Reply:
I've just spoken to him, he's ok with it 👍

Reply:
Yes